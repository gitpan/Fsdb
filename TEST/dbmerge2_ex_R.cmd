prog='dbmerge2'
args='--input TEST/dbmerge_ex_R_b.in cname'
in='TEST/dbmerge_ex_R.in'
cmp='diff -cb '
