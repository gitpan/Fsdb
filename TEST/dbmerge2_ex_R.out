#fsdb -R C cid cname
cid: 11
cname: numanal

cid: 12
cname: os

cid: 10
cname: pascal

#   | dbfilealter -R C
cid: 13
cname: statistics

#   | dbfilealter -R C
#   | dbmerge2 --input TEST/dbmerge_ex_R_b.in cname
